### PR TITLE
feat: add method on app bridge for block bulk download

### PR DESCRIPTION
**Pull request by hochreutenerl**
Meant to work together with Frontify/web-app#895

[Clickup](https://app.clickup.com/t/861n31u4y)

### CODE REVIEW

File: packages/app-bridge/src/react/useAssetBulkDownload.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```
Comment (hochreutenerl) [inline packages/app-bridge/src/react/useAssetBulkDownload.ts:1]: This hook and the tests are basically just a copy of `useBulkDownload`, which can be removed later on

File: .changeset/brown-moose-watch.md
Comment (SamuelAlev) [inline .changeset/brown-moose-watch.md:1]: Beta flow is manual releases so no changeset required, hopefully it will soon be stable `3.0.0` :)

Comment (hochreutenerl) [reply .changeset/brown-moose-watch.md:1]: I removed the file again. @ragi96 FYI

**Comment by SamuelAlev** [comment]
Please reopen your PR, #528 somewhat corrupted the `main` branch, solving it closed the PR.

**Pull request closed without merging**